modules/apps/object/object-service/src/main/java/com/liferay/object/internal/validation/rule/ObjectValidationRuleEngineRegistryImpl.java

```diff
@@ -109,7 +110,14 @@ protected void activate(BundleContext bundleContext) {
 						key);
 				}
 
-				emitter.emit(key);
+				if (FeatureFlagManagerUtil.isEnabled("LPS-188898")) {
+					emitter.emit(key);
+				}
+				else if (!(objectValidationRuleEngine instanceof
+							FunctionObjectValidationRuleEngineImpl)) {
+
+					emitter.emit(key);
+				}
```

Comment:
Hi @mateussgn I believe if we reorder the conditional block like this it'll be more clear, what do you think?

````suggestion
				if (!(objectValidationRuleEngine instanceof FunctionObjectValidationRuleEngineImpl)) {
					emitter.emit(key);
				}
				else if (FeatureFlagManagerUtil.isEnabled("LPS-188898")) {
					emitter.emit(key);
				}
````

Reply:
Yes, @MarinhoFeliphe.
I agree with you!
Thanks for see this!